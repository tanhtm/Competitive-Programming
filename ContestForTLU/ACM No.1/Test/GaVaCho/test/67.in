81395744 168257930
