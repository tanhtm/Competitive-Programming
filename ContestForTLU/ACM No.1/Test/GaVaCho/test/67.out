78662523 2733221
